custom_components/tuya_local/devices/light_sensor.yaml

```diff
@@ -0,0 +1,30 @@
+name: Light sensor
+products:
+  - id: p1632039035856duknw4
+  
+primary_entity:
+  entity: sensor
+  name: Brightness
```

Comment:
There should be no explicit name, so the translations for illuminance can be inherited